src/compiler/checker.ts

```diff
@@ -24019,6 +24022,7 @@ namespace ts {
                 case ExternalEmitHelpers.AsyncDelegator: return "__asyncDelegator";
                 case ExternalEmitHelpers.AsyncValues: return "__asyncValues";
                 case ExternalEmitHelpers.ExportStar: return "__exportStar";
+                case ExternalEmitHelpers.MakeTemplateObject: return "__makeTemplateObject";

```

Comment:
Would it be worth considering a shorter name than `__makeTemplateObject`?

Reply:
@rbuckton Babel uses `_taggedTemplateLiteral` which is two characters longer. If you can think of something by Monday afternoon before I merge this in, I will consider it 😄

Reply:
`__template`? (I like this since it doesn't have any camel-casing)
`__templatize`?
`__intoTemplate`?
`__addRaw`?
`__createTemplateArg`? (not _really_ shorter, I know)